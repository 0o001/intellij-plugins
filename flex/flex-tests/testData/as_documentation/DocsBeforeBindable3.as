package {
public class DocsBeforeBindable3 {

  /**
   *  bla bla bla
   */
  [Bindable("change")]
  
      var f<caret>oo:Boolean;


}
}