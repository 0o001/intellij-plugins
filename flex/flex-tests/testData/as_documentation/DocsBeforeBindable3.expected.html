<div class='definition'><pre>foo: Boolean</pre></div><div class='content'><p>bla bla bla</div>